ref.html

```diff
@@ -156,6 +156,11 @@
       switched companies during edition) or to clarify that an editor is no longer active even if her
       name is still attached to the document out of respect for her past contributions.
     </dd>
+    <dt>w3cid</dt>
+    <dd>
+      Numeric id of the editor profile in W3C; it is used by the
```

Comment:
Not to be picky, but would you mind including where one may avail oneself of such an ID? I for one have no idea whatsoever :)